corehq/apps/geospatial/static/geospatial/js/geospatial_map.js

```diff
@@ -179,17 +183,25 @@ hqDefine("geospatial/js/geospatial_map", [
             }
         };
 
+        var mapControlsModel = function () {
+            'use strict';
+            var self = {};
+
+            self.btnExportDisabled = ko.observable(true);
```

Comment:
I couldn't get the button to disable with this code. I tried using JQuery as well. There must be some hidden (at least from me) reason why this doesn't work. Any help / insight will be appreciated

Reply:
```JS
data-bind="attr: { disabled: btnExportDisabled }"
```

seemed to work